resolwe/flow/serializers/base.py

```diff
@@ -36,7 +36,7 @@ class of child class.
 
     contributor = ContributorSerializer()
     name = serializers.CharField(required=False)
-    slug = serializers.CharField(required=False)
+    slug = serializers.CharField(required=False, allow_null=True)  # Allow null to support slug autogeneration
```

Comment:
Put the comment in a separate line above. And don't forget to put a dot at the end.